components/general/InfoBlock/InfoBlock.tsx

```diff
@@ -1,7 +1,10 @@
 import styles from "./InfoBlock.module.css";
 import { fontSyne, fontSatoshi, fontRobotoMono } from "@/pages/_app";
 import Button from "../../base/Button";
+import EViewPortQuery from "@/constants/viewPortSize";
+import useMediaQuery from "@/hooks/useMediaQuery";
```

Comment:
After tonight discussion, we want to put the useMediaQuery into the index page and the component can use it by transmission, you can refer yunshen's card:https://github.com/taodemy/website/pull/102/files.